specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,537 @@
+- Proposal Name: Multi-Token-Standard
+- Start Date: 2021/07/24
+- Issue(s): #245.
+# Summary
+[summary]: #summary
+
+A standard interface for  a multi token standard that supports fungible, semi-fungible, and tokens of any type, allowing for ownership, transfer, and batch transfer of tokens generally regardless of specific type.
+
+# Motivation
+[motivation]: #motivation
+Having a single contract represent both NFTs and FTs can greatly improve efficiency as demonstrated by Enjin Coin. The ability to make batch requests with multiple asset classes can reduce a many transactions transaciton to a single transaction to trade around both NFTs and FTs that are a part of same token contract.
+
+Having this will also increase Near's ability to work interoperably with other chains. This will reduce the complexity required to represent these emerging asset classes.
```

Comment:
```suggestion
Having this will also increase NEAR's ability to work interoperably with other chains. This will reduce the complexity required to represent these emerging asset classes.
```